x-pack/test/functional/apps/discover/visualize_field.ts

```diff
@@ -171,6 +171,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       await PageObjects.discover.chooseLensChart('Bar vertical stacked');
       await PageObjects.header.waitUntilLoadingHasFinished();
+      await testSubjects.click('TextBasedLangEditor-expand');
```

Comment:
ℹ️ this stabilizes this test. Is irrelevant with the changes made here but as the test failed in this branch I decided to stabilize it here.